deltachat-ios/Controller/ChatListController.swift

```diff
@@ -522,6 +534,75 @@ class ChatListController: UITableViewController {
             view.isHidden = false
         }
     }
+    
+    lazy var accountButtonAvatar: InitialsBadge = {
+        let badge = InitialsBadge(size: 40)
+        badge.setColor(UIColor.lightGray)
+        badge.isAccessibilityElement = false
+        return badge
+    }()
+    
+    private let accountButtonUnreadMessageCounter: MessageCounter = {
+        let view = MessageCounter(count: 0, size: 20)
+        view.backgroundColor = DcColors.unreadBadge
+        view.isHidden = true
+        return view
+    }()
+
+
+    private lazy var accountButton: UIBarButtonItem = {
+        // todo make pretty
+        
+        let rect = CGRect(x: 0, y: 0, width: 70, height: 50)
+        let myView = UIView(frame: rect)
```

Comment:
nitpicking: could you rename it to sth. like containerView? myView is a little bit unusual in this repo.

Reply:
+1, i did the same in the commit i just added.